src/views/InvolvementProfile/components/Membership/components/AdminCard/components/RequestsReceived/index.jsx

```diff
@@ -56,7 +41,7 @@ const RequestsReceived = ({ onAddMember }) => {
           <Fragment key={request.RequestID}>
             <ListItem
               key={request.RequestID}
-              classes={{ secondaryAction: classes.secondaryAction }}
+              classes={{ secondaryAction: styles.secondary_action }}
```

Comment:
any reason why we've switched from camelCase -> snake_case?

Reply:
no specific reason, we can change it to the original way to keep it consistent